### PR TITLE
Feature/la 35703 ds survey category labeling integration for doccano

### PR DESCRIPTION
**Pull request by chihyikuan**
1.Docker version:  20.10.0, build 7287ab3
2.docker build -t  IMAGE_NAME:TAG .  (e.q docker build -t  doccano-test:1.0.0 . )
3.docker container create --name CONTAINER_NAME \
  -e "ADMIN_USERNAME=admin" \
  -e "ADMIN_EMAIL=admin@example.com" \
  -e "ADMIN_PASSWORD=password" \
  -p 8000:8000  IMAGE_NAME:TAG
4.docker container start CONTAINER_NAME

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,6 +6,9 @@ COPY frontend/ /frontend/
 WORKDIR /frontend
 ENV PUBLIC_PATH="/static/_nuxt/"
 
+
```
Comment (amjltc295) [inline Dockerfile:9]: Please remove useless empty lines

File: frontend/components/containers/annotation/TextClassification.vue [hunk 2]
```diff
@@ -6,7 +6,7 @@
   >
     <v-card-title>
       <multi-class-classification
-        :labels="items"
+        :labels="itemsl"
```
Comment (amjltc295) [inline frontend/components/containers/annotation/TextClassification.vue:9]: Could you rename it as itemsSecondLevel?

File: frontend/components/containers/annotation/TextClassification.vue [hunk 3]
```diff
@@ -21,14 +21,38 @@ import Vue from 'vue'
...
   components: {
     MultiClassClassification
   },
-
   computed: {
-    ...mapState('labels', ['items']),
+    // ...mapState('labels', ['items']),
+    ...mapState('labels', {
+      items(state) {
+        console.log('level1', state)
```
Comment (amjltc295) [inline frontend/components/containers/annotation/TextClassification.vue:32]: Please remove debugging logs

**Review by amjltc295 — CHANGES_REQUESTED** [review]

**Comment by chihyikuan** [comment]
Done modify reviewed issue. @amjltc295

**Review by amjltc295 — APPROVED** [review]

**Comment by amjltc295** [comment]
We have to fix https://github.com/doccano/doccano/pull/1197

**Pull request merged**